math-blocks/math-blocks
# Use 'text' nodes for non-delimiter glyphs

## Summary

This greatly reduces the size of the generated SVG.

## Test Plan

- open http://localhost:3000/#/editor
- use the inspect to check that the alphanumeric glyphs rendered as `text` nodes
- switch between different fonts to see that the fonts display correctly
